pkg/policy/validate.go

```diff
@@ -20,9 +22,41 @@ import (
 	rbacv1 "k8s.io/api/rbac/v1"
 	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
 	log "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+//check for forward slash
+func JSONPatchPathHasForwardSlash(patch string) error {
```

Comment:
change the function name as `validateJSONPatchPathForForwardSlash` and the comment over the function as `validateJSONPatchPathForForwardSlash checks for forward slash`